codebuild/spec/buildspec_address_sanitizer.yml

```diff
@@ -0,0 +1,141 @@
+version: 0.2
+
+# This buildspec runs on an Ubuntu22 image. That configuration is a property of
+# the codebuild job itself.
+
+# Codebuild's matrix jobs have non-differentiated names so use batch-list
+# instead.
+
+# Parameter motivation
+
+# COMPILERS
+# We run asan on both gcc and clang because of different features sets for their
+# address sanitizers. Specifically there was a case where GCC was able to detect
+# a memcpy-param-overlap that Clang did not.
```

Comment:
Does this apply to the undefined behavior sanitizers in addition to the address sanitizers? Are we gaining anything by running both?

Reply:
Address sanitizer and undefined behavior sanitizers are two different checks. I will update the comments to make it more accurate. We run both gcc and clang on both asan check and ubsan check.